contents/blog/equity_explainer.md

```diff
@@ -0,0 +1,99 @@
+---
+date: 2022-10-20
+title: "Rising tides: our employee-friendly share options scheme"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Inside PostHog", "Startups", "Guides"]
+author: ["grace-mckenzie"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+If you’ve ever worked for a startup, or read about a Unicorn tech company turning employees into multi-millionaires overnight, chances are you've heard of equity.
+
+Equity is the amount of ownership of a corporation held by a person or entity. This ownership is usually described as either a percentage of the total equity of the business or, at later stages, a finite number of shares. 
+
+The majority of tech start-ups offer equity as part of their total compensation, normally in the form of share “options”. The upside potential of these options is meant to compensate for the more gruelling (and in my opinion, rewarding) work of growing a start-up company from founding to IPO (more on that later).
+
+When we set out to design a share options scheme for PostHog, we decided to prioritize employee-friendliness for a number of reasons:
+
+1. Many of us had personally been burned by unfriendly share option schemes when working at previous companies.
+
+2. It helps us to attract talent by making our share options massively more valuable in practical terms, not just the pure $ amount (which in many ways may be the least important thing).
+
+3. It's part of our culture to expect high performance but reward people well and ensure they are genuinely able to share in PostHog's growth. Having a good share options scheme helps people feel more bought in.
+
+4. We also feel that it's just the right thing to do!
+
+## How share options work
+
+When a startup company is incorporated, the founders decide on how to divide ownership between them by (quite literally) choosing a random number of shares to create out of thin air and distributing them to each other in some proportion. 
+
+In the common two-founder model, most start-ups distribute the shares as a 50/50 split between them, accounting for 100% ownership of the business. The shares are "purchased" for a nominal cost like a ten-thousandth of a cent because the value of a newly minted start-up company is effectively nothing.
```

Comment:
```suggestion
In the common two-founder model, most startups distribute the shares as a 50/50 split between them, accounting for 100% ownership of the business. The shares are "purchased" for a nominal cost like a ten-thousandth of a cent because the value of a newly minted start-up company is effectively nothing.
```